scripts/vic_utils

```diff
@@ -59,37 +61,202 @@ def main():
+                                    default=False)
+    grid_params_parser.add_argument("--vegparam_albedo",
+                                    type=str,
+                                    help="Flag indicating whether veg "
+                                         "parameter file contains albedo "
+                                         "values (default=False)",
+                                    default=False)
+    grid_params_parser.add_argument("--lai_src",
+                                    type=str,
+                                    help="Where to take lai values from; "
+                                         "can be one of: \'FROM_VEGLIB\', "
+                                         "\'FROM_VEGPARAM\' "
+                                         "(default=\'FROM_VEGLIB\')",
+                                    choices=['FROM_VEGLIB', 'FROM_VEGPARAM'],
+                                    default="FROM_VEGLIB")
+    grid_params_parser.add_argument("--fcan_src",
+                                    type=str,
+                                    help="Where to take fcanopy values from; "
+                                         "can be one of: \'FROM_DEFAULT\', "
+                                         "\'FROM_VEGLIB\', \'FROM_VEGPARAM\'; "
+                                         "\'FROM_DEFAULT\' = set them to 1.0; "
+                                         "(default=\'FROM_DEFAULT\')",
+                                    choices=['FROM_DEFAULT',
+                                             'FROM_VEGLIB', 'FROM_VEGPARAM'],
+                                    default="FROM_DEFAULT")
+    grid_params_parser.add_argument("--alb_src",
+                                    type=str,
+                                    help="Where to take albedo values from; "
+                                         "can be one of: \'FROM_VEGLIB\', "
+                                         "\'FROM_VEGPARAM\' "
+                                         "(default=\'FROM_VEGLIB\')",
+                                    choices=['FROM_VEGLIB', 'FROM_VEGPARAM'],
+                                    default="FROM_VEGLIB")
+    grid_params_parser.add_argument("--lake_profile",
+                                    type=str,
+                                    help="Flag indicating whether lake "
+                                         "parameter file contains lake "
+                                         "depth-area relationship "
+                                         "(default=False)",
+                                    default=False)
+    grid_params_parser.add_argument("--carbon",
+                                    type=str,
+                                    help="Flag indicating whether state "
+                                         "file contains carbon-cycle "
+                                         "variables (default=False)",
+                                    default=False)
```

Comment:
for the options that are just boolean flags, lets do:

```Python
type=bool,
default=False,
action='store_true', 
help='...')

Reply:
Well, I had to roll back the changes that I made in response to your suggestion - they broke the travis build.  I've kept the change in type to `bool`.  But I've not added the `action=store_true` or the move of `default=False` to be second in the order.

Reply:
What was the error you were getting? This should work.

Reply:
It had a problem with `type` not recognized or something like that.  I don't have the error accessible anymore.  The error message doesn't make sense of course, since `type` is still there in the same position in the arg list.

Can you look into that issue separately, and deal with it in some other PR?  It's not related to the change I'm making.

Reply:
Actually, I found my travis build log.  The error was the same in python 2.7 and 3.4:
TypeError: __init__() got an unexpected keyword argument 'type'

Reply:
Ok. Take a look at the docs. Looks like you just need the `action='store_true'`. No default/dtype is required.

https://docs.python.org/3/library/argparse.html#action

Reply:
OK, I replaced the `type=bool` and `default=False` arguments with `action='store_true'`.